onnxruntime/core/providers/dml/DmlExecutionProvider/src/PooledUploadHeap.h

```diff
@@ -32,6 +32,7 @@ namespace Dml
     private:
         static constexpr size_t c_minChunkSize = 1024 * 1024; // 1MB
         static constexpr size_t c_allocationAlignment = 512; // In bytes; as per D3D12 requirement for buffers
+		static constexpr size_t c_maxChunkSize = 0xffff0000; // ~4 GiB dx12 limitation for DX12 CPU visible resource
```

Comment:
```suggestion
        static constexpr size_t c_maxChunkSize = 0xffff0000; // ~4 GiB dx12 limitation for DX12 CPU visible resource
```
Avoid tabs.
![image](https://github.com/user-attachments/assets/41bf0036-986e-42f4-b6ac-c605611af6e2)